lib/kanban/key_value_store.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Kanban.KeyValueStore do
+  @moduledoc """
+  The KV-Store KeyValueStore
+  """
+
+  use GenServer
+
+  def start_link(state) do
+    GenServer.start_link(__MODULE__, state, name: __MODULE__)
+  end
+
+  @impl GenServer
+  def init(state), do: {:ok, state}
+
+  @impl GenServer
+  def handle_call({:put, key, value}, _from, state) do
+    state = Map.update(
+      state,
+      key,
+      [value],
+      fn values -> [value | values] end
```

Comment:
Просто замечу, что можно использовать function captures:
```suggestion
      &[&1 | values]
```